patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -1,6 +1,38 @@
 --- ../src-base/minecraft/net/minecraft/entity/EntityLivingBase.java
 +++ ../src-work/minecraft/net/minecraft/entity/EntityLivingBase.java
-@@ -201,10 +201,11 @@
+@@ -1,7 +1,5 @@
+ package net.minecraft.entity;
+ 
+-import com.google.common.base.Objects;
+-import com.google.common.collect.Maps;
+ import java.util.Collection;
+ import java.util.ConcurrentModificationException;
+ import java.util.Iterator;
+@@ -9,7 +7,15 @@
+ import java.util.Map;
+ import java.util.Random;
+ import java.util.UUID;
++
+ import javax.annotation.Nullable;
++
++import org.apache.logging.log4j.LogManager;
++import org.apache.logging.log4j.Logger;
```

Comment:
You have unused imports

Reply:
This is a patch, no touchy imports.

Reply:
So do I have to remove the unused imports or not?